api/models/site.js

```diff
@@ -165,6 +165,8 @@ module.exports = (sequelize, DataTypes) => {
       afterValidate,
       validationFailed,
     },
+    timestamps: true,
+    paranoid: true,
```

Comment:
interesting, didn't even know that this was part of sequelize. neat!

Reply:
Yes, its realllllly helpful to not have to write this logic ourselves!